src/include/uapi/abi.h

```diff
@@ -52,7 +52,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 4
+#define SOF_ABI_MINOR 5
```

Comment:
increasing the ABI twice in a PR seems weird. Is this intentional?

Reply:
Yes, there are 2 separate commits altering the ABI